c++/src/kj/compat/http.h

```diff
@@ -748,7 +748,12 @@ class HttpClient {
     kj::Own<kj::AsyncIoStream> connection;
   };
 
-  virtual ConnectRequest connect(kj::StringPtr host, const HttpHeaders& headers);
+  struct ConnectSettings {
+    bool useTls;
```

Comment:
All settings should have defaults so people don't have to manually initialize them.

```suggestion
    bool useTls = false;
```

Then you can change all the places where you've written `{ .useTls = false }` to be just `{}`.

(That in turn should, by coincidece, help fix the compile errors on Windows/MSVC, where designated initializers are not supported in C++14 mode, which is what Cap'n Proto currently uses.)